learn/framework/templates.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Templates
+description: Substrate is a powerful blockchain development framework that provides many ready-to-use templates for building parachains, nodes and more, and allowing product teams to quickly and easily build Appchains.
+---
+
+# Templates {: #templates } 
+
+## Starting Point {: #starting-point } 
+
+To start building an Appchain, there are some useful templates to kick-start the development process:
+
+- **[Node Template](https://github.com/substrate-developer-hub/substrate-parachain-template){target=_blank}** - This node template contains a default configuration to quickly run and operate what would represent a single node in a blockchain
+- **[Front end Template](https://github.com/substrate-developer-hub/substrate-parachain-template){target=_blank}** - This template contains a web client developed using ReactJS and PolkadotJs API to interact with your Appchain with an easy-to-grasp UI
+- **[Appchain Template](https://github.com/substrate-developer-hub/substrate-parachain-template){target=_blank}** - This template contains a default configuration to run an Appchain (a Parachain within the Polkadot ecosystem)
+
+## Running and Testing the Network {: #running-and-testing-the-network } 
+
+Becoming a Tanssi ContainerChain brings along the benefits of being part of a mature ecosystem, leveraging the inherited security of the relay chain and the ability to communicate and share assets and data with other siblings ContainerChains and Parachains.
```

Comment:
This is the first mentioning of a ContainerChain, maybe link to where someone can find a definition or explanation of what a ContainerChain is

I think because Tanssi is different than Moonbeam, perhaps you need different guidelines on what to capitalize, like "Parachain", "Appchain", "ContainerChain", "Relay Chain", etc. 

I know I've made comments throughout this PR on not capitalizing some of these words, but maybe just take some time to consider it and come up with a standard for Tanssi to use and whatever you decide just be consistent and stick with it